pwsh/Helpers.ps1

```diff
@@ -0,0 +1,128 @@
+$ALLOWED_ASSET_EXTENSIONS = @{
```

Comment:
a file-level comment should be added noting these are support routines for the other functions defined in files